profilingdemo/source/cpp/CMakeLists.txt

```diff
@@ -35,6 +35,6 @@ else (NOT DEFINED KAA_SDK_PATH)
 endif (NOT DEFINED KAA_SDK_PATH)
 
 add_executable(demo_client src/KaaDemo.cpp)
-target_link_libraries(demo_client kaacpp)
+target_link_libraries(demo_client kaacpp ${Boost_FILESYSTEM_LIBRARY})
```

Comment:
No, it doesn't. There might be a list of libraries in this variable and we want it to expand into a space-separated list of items